sdk/attestation/Azure.Security.Attestation/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
 # Release History
-## 1.0.0-alpha.1 (2020-12-08)
+## 1.0.0-beta.1 (2020-12-08)
```

Comment:
You should leave -alpha there because it's what shipped and add an extra  `## 1.0.0-beta.1 (Unreleased)` line above it for the next release like the auto-increment tool would normally have done.

Reply:
Fixed in current iteration. TYVM.